src/core/public/application/ui/app_container.tsx

```diff
@@ -21,6 +21,7 @@ import { AppLeaveHandler, AppStatus, AppUnmount, Mounter } from '../types';
 import { AppNotFound } from './app_not_found_screen';
 import { ScopedHistory } from '../scoped_history';
 import './app_container.scss';
+import { APP_WRAPPER_CLASS } from '../../../../../src/core/public';
```

Comment:
1. importing within core from core's index file can introduce cyclic imports and should be avoided.
2. You can safely import directly from `src/core/utils` here

=>

```
import { APP_WRAPPER_CLASS } from '../../../utils';
```